src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -86,6 +88,7 @@ class OPENVINO_GENAI_EXPORTS GenerationConfig {
     bool ignore_eos = false;
     size_t min_new_tokens = 0;
     bool echo = false;
+    size_t logprobs = 0;
```

Comment:
BTW, why is it not `top_logprobs` ?

Reply:
I'm opened to suggestions. I was trying to follow OpenAI API naming, and it goes with `logprobs` on one endpoint and `top_logprobs` on another one. Both essentially mean the same, so if you prefer `top_logprobs` then I can change it.